rpc-rs/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "wasmbus-rpc"
-version = "0.10.1"
+version = "0.10.2"
```

Comment:
Request to bump to `0.11.0` even though it'll require some upgrades, @stevelr may have some things he wants to put into the new release